.github/workflows/release.yml

```
@@ -1,183 +1,196 @@
 
-          | Field | Value |
-          |---|---|
-          | Version | \`${VERSION}\` |
-          | versionCode | \`${VERSION_CODE}\` |
-          | SHA256 | \`${CHECKSUM_VAL}\` |
+          NOTES="See [README](https://github.com/${{ github.repository }}) 👀
+- Release \\`${VERSION}\\` (versionCode \\`${VERSION_CODE}\\`)
 
-          See [README.md](https://github.com/${{ github.repository }}/blob/${RELEASE_TAG}/README.md) for full installation, configuration, and recovery instructions."
+## What's Changed
+* Automated module build + packaging for ${VERSION}.
```

actionlint (1.7.11)

[error] could not parse as YAML: did not find expected alphabetic or numeric character

(syntax-check)